docker/build-arch-package.Dockerfile

```
@@ -0,0 +1,82 @@
+      python-pyperclip \
+      python-toml \
+      python-wcwidth \
+      python-yaml \
+  && pacman -Scc --noconfirm
+
+# PyInstaller is not in the official Arch repos (only AUR), so install it with pip
+# into the system interpreter. PEP 668 marks Arch's Python as externally managed;
+# --break-system-packages is the supported override inside this throwaway build
+# image. makepkg is run with --nodeps (see scripts/build_and_package_arch.py)
+# because this build image - not pacman - provisions the build toolchain.
+RUN python -m pip install --no-cache-dir --break-system-packages pyinstaller
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)